bin/contact-search-backfill

```diff
@@ -1,14 +1,16 @@
 #!/usr/bin/env python
 import click
```

Comment:
Click should be moved down since `monkey.patch_all()` should be the first thing.